app/schema.py

```diff
@@ -52,6 +52,28 @@ class MentorUpdate(ExtraForbid):
     accepting_new_mentees: Optional[bool]
 
 
+class MentorOptions(ExtraForbid):
```

Comment:
Greatly appreciate the buildout on this as we are now a step closer to being able to work with Mentor Data!